### PR TITLE
Tweaks

### PR DESCRIPTION
**Pull request by aviflax**
Yes, really :trollface: 

I recommend reviewing each commit individually.

Thanks!

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -53,17 +53,20 @@ jobs:
...
            # Puppeteer. So it’s a good idea to ensure that the packaged renderer actually works.
+           # The JVM component’s ability to invoke the packaged renderer is also susceptible to
+           # breakage, so here we try to invoke the tool to render a diagram just as a user would.
            name: Test packaged renderer
            command: |
              mkdir test-renders
-             cat test/data/structurizr/express/diagram_valid_messy.yaml \
-               | target/pkg/renderer/render-linux --verbose > test-renders/1.png
+             target/pkg/linux/fc4/fc4 render \
+               test/data/structurizr/express/diagram_valid_messy.yaml > test-renders/1.png
```
Comment (sgerrand) [inline .circleci/config.yml]: 🙋 It looks like the rendered file is empty:
```
Uploading /root/project/tool/test-renders to root/project/tool/test-renders
Uploading /root/project/tool/test-renders/1.png: SKIPPED, file is empty
```
https://circleci.com/gh/FundingCircle/fc4-framework/728

Is that intentional?

File: tool/renderer/render.js [hunk 2]
```diff
@@ -118,7 +118,16 @@ function prepYaml(yaml) {
   return sepLoc >= 0 ? yaml.substring(sepLoc) : `---\n${yaml}`;
 }
 
-async function main(debugMode) {
+function parseArgs() {
+  const args = process.argv.join();
+  return {
+    debugMode: args.includes('--debug')
```
Comment (sgerrand) [inline tool/renderer/render.js]: 👏

File: tool/bin/pkg-all [hunk 3]
```diff
@@ -1,7 +1,9 @@
-#!/bin/sh
+#!/bin/bash
 ## Run this from <project-root>/tool/
 set -e
 
+[ "$GIT_HEAD_SHA" ] || { echo 'GIT_HEAD_SHA is not set!' && exit 1; }
```
[the remark this answers is not in the document]
Comment (aviflax) [reply tool/bin/pkg-all]: Good to know, thank you!

File: .circleci/config.yml [hunk 4]
```diff
@@ -53,17 +53,20 @@ jobs:
...
            command: |
+             export GIT_HEAD_SHA=$(git rev-parse --short HEAD)
              bin/pkg-all
              mkdir -p /workspace/packages
              mv target/pkg/*.gz /workspace/packages/
        - run:
            # Packaging any node app with pkg can be tricky; that’s doubly true for an app that uses
            # Puppeteer. So it’s a good idea to ensure that the packaged renderer actually works.
+           # The JVM component’s ability to invoke the packaged renderer is also susceptible to
+           # breakage, so here we try to invoke the tool to render a diagram just as a user would.
```
[the remark this answers is not in the document]
Comment (aviflax) [reply .circleci/config.yml]: 😊

**Pull request merged**